src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx

```diff
@@ -265,8 +285,8 @@ class UploadDetachedFileValidation extends React.Component {
 		validationBox = <ValidateDataFileContainer type={type} data={this.state.jobResults}/>;
 		if (!this.state.headerErrors && this.state.validationFinished) {
 			validationBox = <ValidateValuesFileContainer type={type} data={this.state.jobResults} setUploadItem={this.uploadFile.bind(this)} updateItem={this.uploadFile.bind(this)} published={this.state.published}/>;
-			if (this.state.published){
-				// Submission is published and cannot be re-published
+			validationButton = <button className='pull-right col-xs-3 us-da-button' onClick={this.openModal.bind(this)}>Publish</button>;
```

Comment:
why would it always be overwritted? If it is not published then it will just say publish.